cameraServer.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+import json
+import time
+import sys
+import numpy as np
+import cv2
+import math
+
+#Magic Numbers
+lowerGreen = (38, 125, 100) #Our Robot's Camera
+higherGreen = (52, 255, 180)
+sample_lowerGreen = (30, 177, 80) #FRC sample images
+sample_higherGreen = (150, 255, 255)
+lowerTapeColour = (0, 0, 0) #TODO: Ground Tape filter colours
+upperTapeColour = (360, 255, 255) #TODO: Ground Tape filter colours 
+minContourArea = 10
+angleOffset = 13
+rightAngleSize = -14.5
+leftAngleSize = -75.5
+screenSize = (320, 240)
+
+#Initialisation
+configFile = "/boot/frc.json"
+
+class CameraConfig: pass
+
+team = None
+server = False
+cameraConfigs = []
+
+"""Read single camera configuration."""
+def readCameraConfig(config):
+    cam = CameraConfig()
+
+    # name
+    try:
+        cam.name = config["name"]
+    except KeyError:
+        parseError("could not read camera name")
+        return False
+
+    # path
+    try:
+        cam.path = config["path"]
+    except KeyError:
+        parseError("camera '{}': could not read path".format(cam.name))
+        return False
+
+    cam.config = config
+
+    cameraConfigs.append(cam)
+    return True
+
+"""Report parse error."""
+def parseError(str):
```

Comment:
[str](https://docs.python.org/3/library/functions.html#func-str) is a built in and should not be used as a function argument